src/plone/api/content.py

```diff
@@ -523,27 +523,30 @@ def get_view(name=None, context=None, request=None):
     # available, because the __init__ of said view will contain
     # errors in client code.
 
-    # Get all available views...
-    sm = getSiteManager()
-    available_views = sm.adapters.lookupAll(
-        required=(providedBy(context), providedBy(request)),
-        provided=Interface,
-    )
-    # and get their names.
-    available_view_names = [view[0] for view in available_views]
-
-    # Raise an error if the requested view is not available.
-    if name not in available_view_names:
-        raise InvalidParameterError(
-            "Cannot find a view with name '{name}'.\n"
-            "Available views are:\n"
-            "{views}".format(
-                name=name,
-                views="\n".join(sorted(available_view_names)),
-            ),
+    try:
+        adapter = getMultiAdapter((context, request), name=name)
+    except Exception: 
```

Comment:
I think this should be `except ComponentLookupError`, otherwise you risk to return a wrong message.

Reply:
Thank you for pointing it out! I have made the changes accordingly.